RecoParticleFlow/PFClusterProducer/plugins/SimMappers/ComputeClusterTime.h

```diff
@@ -0,0 +1,128 @@
+#ifndef RecoParticleFlow_PFClusterProducer_ComputeClusterTime_h
+#define RecoParticleFlow_PFClusterProducer_ComputeClusterTime_h
+
+// user include files
+#include <vector>
+
+
+// functions to select the hits to compute the time of a given cluster
+// start with the only hits with timing information
+// average among the hits contained in the chosen time interval
+
+// N.B. time is corrected wrt vtx-calorimeter distance 
+// with straight line and light speed hypothesis
+// for charged tracks or heavy particles (longer track length or beta < 1) 
+// need to correct the offset at analysis level
+
+
+
+namespace hgcalSimClusterTime {
```

Comment:
the CMS style is for lower case letters for the namespaces
this is a common style in the industry as well